domain/domain.go

```diff
@@ -645,31 +665,26 @@ func (do *Domain) updateStatsWorker(ctx context.Context, lease time.Duration) {
 			if err != nil {
 				log.Error("[stats] load histograms fail: ", errors.ErrorStack(err))
 			}
+		case <-gcStatsTicker.C:
+			if !owner.IsOwner() {
+				continue
+			}
+			err := statsHandle.GCStats(do.InfoSchema())
```

Comment:
Can we make sure that ddl is working well before executing gc?

Reply:
Why do we need to make sure that?

Reply:
An outdated infoschema is horrible, it may lack some table info so we think it has been deleted....

Reply:
I think that we can make sure that, 10 lease before `PrevLastUpdateVersion` is quite long...